### ISSUE TITLE
DesignTime issue with File Scoped Namespace

### ISSUE THREAD

**Issue by lbrecheisen** [max-ieremenko/ServiceModel.Grpc#73]
**Describe the bug**
C# 10 file scoped namespaces cause build error with source code generation.

`error CS1106: Extension method must be defined in a non-generic static class`

**To Reproduce**
```c#
using MyNamespace.Services;
using ServiceModel.Grpc.DesignTime;

namespace MyNamespace.Grpc;

[ExportGrpcService(typeof(MyService), GenerateAspNetExtensions = true)]
internal static partial class MyGrpcServices
{

}
```

**Expected behavior**
File scoped namespaces should work just like traditional namespaces during source code generation.

**Environment**
 - OS: Windows 11
 - .NET 6

**Comment by max-ieremenko** [comment]
thanks for the bug, yes the code generator does not know about "scoped namespaces"

**Comment by max-ieremenko** [comment]
Fix released in version 1.4.4.

**Comment by lbrecheisen** [comment]
> Fix released in version 1.4.4.

Thanks! Great library btw.
